.github/workflows/check-changeset.yml

```diff
@@ -0,0 +1,135 @@
+
+                    for (const comment of comments.data) {
+                        console.log("COMMENT_OWNER: " + comment.user.login);
+
+                        // Identify the changeset comment by its heading.
+                        if (comment.body.includes("🦋 Changeset detected") || comment.body.includes("⚠️ No Changeset found")) {
+                            console.log("COMMENT_ID_TO_DELETE: " + comment.id);
+
+                            // Remove the changeset comment using the comment ID.
+                            await github.issues.deleteComment({
+                                owner: REPO_OWNER,
+                                repo: REPO_NAME,
+                                comment_id: comment.id,
+                            });
+                        }
+                    }
+
+            - name: 💬 Add Changeset Comment
+              uses: actions/github-script@v3.1.0
+              with:
+                github-token: ${{ env.GH_TOKEN }}
+                script: |
+                    const fs = require('fs');
+                    const PR_NUMBER = Number(fs.readFileSync('./PR_NUMBER', 'utf8').trim());
+                    const REPO_OWNER = context.repo.owner;
+                    const REPO_NAME = context.repo.repo;
+
+                    const files = await github.pulls.listFiles({
+                        owner: REPO_OWNER,
+                        repo: REPO_NAME,
+                        pull_number: PR_NUMBER,
+                    });
+
+                    const CHANGED_FILES = files.data.map(file => file.filename);
+                    console.log("CHANGED_FILES_URL: https://api.github.com/repos/" + REPO_NAME + "/pulls/" + PR_NUMBER + "/files");
+                    console.log("CHANGED_FILES:", CHANGED_FILES);
+
+                    const CHANGES_COUNT = CHANGED_FILES.filter(filename => /^\.changeset\/.*\.md$/.test(filename)).length;
+                    console.log("CHANGES_COUNT:", CHANGES_COUNT);
+
+                    let COMMENT;
+                    if (CHANGES_COUNT > 0) {
+                        console.log("Changeset detected");
+                        COMMENT = `<h3>🦋 Changeset detected</h3><p><b>The changes in this PR will be included in the next version bump.</b></p><p>Not sure what this means? <a href="https://github.com/changesets/changesets/blob/master/docs/adding-a-changeset.md">Click here to learn what changesets are</a>.</p>`;
+                    } else {
+                        console.log("No changeset detected");
+                        COMMENT = `<h3>⚠️ No Changeset found</h3>Merging this PR will not cause a version bump for any packages. If these changes should not result in a new version, you're good to go.</p><p><b>If these changes should result in a version bump, you need to add a changeset.</b></p><a href="https://github.com/changesets/changesets/blob/master/docs/adding-a-changeset.md">Click here to learn what changesets are, and how to add one</a>`;
```

Comment:
It was observed that developers do not pay much attention to this bot message and always reach out to the team to learn what changeset is and how to add changeset. 

To avoid that, we may personalize this message with a salutation to the pr owner, and a link to our release docs. WDYT?

Reply:
Added repo's release doc link. Will do other improvements as time permits.
d86ff68cc484d08b5fa08f8c8dac7b113a223373

Reply:
Ack